pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -221,11 +222,25 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 	// Set awsAccountClaim.Spec.AwsAccountOU
 	if accountClaim.Spec.AccountOU == "" || accountClaim.Spec.AccountOU == "ROOT" {
 
+		// Determine if in fedramp env
+		configMap, err := utils.GetOperatorConfigMap(r.client)
+		if err != nil {
+			log.Error(err, "failed retrieving configmap")
+		}
+		ifFedramp, err := utils.IsFedramp(configMap)
+		if err != nil {
+			log.Error(err, "Unable to verify if cluster is fedramp")
+		}
+		awsRegion := "us-east-1"
+		if ifFedramp {
+			awsRegion = "us-gov-east-1"
+		}
```

Comment:
Wouldn't we want to return an error if we can't read the config map and cancel reconcilation?
I see these lines duplicated multiple times in the PR, probably worth encapsulating in their own function?